core/commands/generate_repository_partials/clustrix_parser.rb

```diff
@@ -64,17 +66,28 @@ def self.get_clustrix_release_versions(config, private_key, link_name)
     end
   end
 
+  def self.get_platform_version_by_link(link)
+    link.match(/el\d+.tar.bz/i).to_s.split(/[^\d]/).join
+  end
+
   def self.generate_clustrix_release_info(platforms, release_info, product_name)
     platforms.map do |platform_and_version|
       platform, platform_version = platform_and_version.split('_')
```

Comment:
This line must be reworked when initial list will be shifted from the `centos_7` to `centos` pattern.